### DISCUSSION TITLE
Improve Rich Text Editing In Dark Mode

### DISCUSSION THREAD

**Discussion by atrent0815** [burkybang/SuiteAdvanced-Feedback#121] [Ideas]
While using dark mode, it is very hard to read and edit rich text fields. It would be awesome to have the same dark mode option for rich text as provided by the code editor settings

**Comment by burkybang** [comment]
Can you provide a screenshot and your NetSuite version (2023.2 or 2024.1)?

**Reply by atrent0815** [reply]
No problem - I am still on 23.2 and attached is a rich text field (comparing souce code on/off - if I turn on source code it turns the field to dark mode) 
![source code in rich text](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/117312828/727a5fcc-f438-4830-88b9-c083f86f1241)
![native editor](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/117312828/5ae3bbd0-1ba7-42a8-80f8-daefdd2a163c)

**Reply by burkybang** [reply]
What's difficult about it? Is it just that the formatted view is not dark? If so, this is intended. In my opinion, it's better that the formatted view resembles exactly what will be output.

**Reply by atrent0815** [reply]
![Default text color](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/117312828/eda00492-e033-4f3f-abbc-f862347a0bfc)
This screenshot might help explain better - the default text color for rich text fields is very faint grey when you turn on dark mode instead of black when dark mode is off. As a personal preference, the entire white space is very bright when you are used to dark mode everywhere else, so I would prefer more background color options. However, I understand what you are saying about respecting the output so I could understand if you only could address the font color. Thank you!!

**Reply by burkybang** [reply]
@atrent0815 Fixed in latest version that's out now!

**Reply by atrent0815** [reply]
That helped a lot, thanks!
